### PR TITLE
TST: Use Actions

### PR DESCRIPTION
**Pull request by pllim**
As advertised. Not sure if CI will pass because Travis CI last run failed.

See https://github.com/pllim/astropy-benchmarks/actions

### CODE REVIEW

File: .github/workflows/ci_workflows.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        python-version: '3.8'
+    - name: Install
+      run: |
+        sudo apt-get install mlocate
+        python -m pip install scipy matplotlib asv
+        asv machine --machine Actions --os unknown --arch unknown --cpu unknown --ram unknown
+    - name: Test with stable astropy
+      run: |
+        python -m pip install astropy=={{ matrix.astropy_version }}
+        asv run -q
```
Comment (pllim) [inline .github/workflows/ci_workflows.yml:36]: `You have an error in your yaml syntax on line 36`

Need help from someone with YAML-fu.

**Comment by astrofrog** [comment]
Will take a look shortly, thanks! - note that there is also a different Travis config on the results branch.

**Comment by pllim** [comment]
I can't get the `if` right. Maybe you know how to fix... https://github.com/pllim/astropy-benchmarks/actions/runs/362426378

**Comment by pllim** [comment]
> results branch

Does that branch run CI too?

**Comment by astrofrog** [comment]
Yes it runs a CI to publish to gh-pages

**Comment by pllim** [comment]
Re: gh-pages -- Have something theoretical at astropy/astropy-dashboard#29 but it doesn't look pretty. Ideas welcome.

**Comment by pllim** [comment]
Re: gh-pages -- Looks like @adrn got something to work using a third-party app as well. https://github.com/adrn/astropy-sustainability/blob/239458ccd95649d064fe925dc4cabeafb7025800/.github/workflows/run.yml#L63

**Pull request closed without merging**